docs/canonicalk8s/charm/howto/install/install-terraform.md

```diff
@@ -21,88 +21,48 @@ controller. Choose one of the options outlined in the
 
 The Terraform deployment is done using a root module that specifies the
 Juju model to deploy the submodules to. The root module also references
-the k8s charm and the k8s-worker charm that each live in a separate child
-module.
+the k8s-bundle module which helps to build the juju model. 
 
 ### Root module
 <!-- TODO replace this section once we have a Juju ground up module -->
 
 The root module ensures that Terraform is aware of the `juju_model`
-dependency of the charm module. Additionally, it contains the path to the k8s
-and k8s-worker child modules:
+dependency of the charm module. Additionally, it contains the path to the
+k8s-bundle child modules:
 
 Example `main.tf`:
 
 ```hcl
-provider "juju" {}
-
-resource "juju_model" "my_model" {
-  name = "juju-myk8s"
-}
-
 module "k8s" {
-  source  = "git::https://github.com/canonical/k8s-operator//charms/worker/k8s/terraform?ref=main"
-
-  model   = juju_model.my_model.name
-  channel = var.channel
-  units = 3
-}
-
-module "k8s-worker" {
-  source  = "git::https://github.com/canonical/k8s-operator//charms/worker/terraform?ref=main"
-  model   = juju_model.my_model.name
-  channel = var.channel
-  units = 2
+  source  = "git::https://github.com/canonical/k8s-bundles//terraform?ref=main"
+  model   = {
+    name  = "my-canonical-k8s-model"
+    cloud = "prod-k8s-openstack"
+  }
+  cloud_integration = "openstack"
+  manifest_yaml = "/path/to/manifest.yaml"
 }
 ```
 
-Example `variables.tf`:
+Define your `manifest.yaml` based on the requirements for your deployment. 
+Specific charm configuration options can be found on charmhub.io for charms
+[k8s] and [k8s-worker].
 
-```hcl
-variable "channel" {
-  description = "K8s deployment channel"
-  type        = string
-}
-```
-
-Example `integrations.tf`:
+Example `manifest.yaml`:
 
 ```hcl
```

Comment:
please remove the hcl here. Either change it yaml or leave blank as it is making a weird rendering on RTD
![image](https://github.com/user-attachments/assets/dd15c936-cbb2-4e28-ab4b-2e32ede295ff)